plugins/rest/komand_rest/util/util.py

```diff
@@ -61,6 +63,31 @@ def first(sequence, default=""):
     return next((x for x in sequence if x), default)
 
 
+def check_headers_for_urlencoded(headers: Dict[str, str]) -> bool:
```

Comment:
If that one can be `None` then it should be added in the type-hint as well 😃 
```suggestion
def check_headers_for_urlencoded(headers: Union[Dict[str, str], None]) -> bool:
```